File: HereForYou/Entities/SiteVisit.cs
using System;
using LinqToDB.Mapping;

namespace HereForYou.Entities
{
    [Table("SiteVisit", IsColumnAttributeRequired = false)]
    public class SiteVisit
    {
        [Identity, PrimaryKey]
        public int Id { get; set; }

        public DateTime VisitDate { get; set; }
        public string WorkerName { get; set; }
        public bool BibleGiven { get; set; }
        public int SiteId { get; set; }

        public string TeamMembers { get; set; }
        public string Notes { get; set; }
        public bool GiftsGiven { get; set; }
        public bool LoginGiven { get; set; }
        public bool Prayer { get; set; }

        public string EnglishAbility { get; set; }
        public string Age { get; set; }
        public bool FollowUpRequired { get; set; }

        public bool DressedInappropriately { get; set; }
        public bool InadequateLighting { get; set; }
        public bool CanNotLeave { get; set; }
        public bool SleepingOnSite { get; set; }
        public bool BeingMovedAround { get; set; }
        public bool DoesntHaveDocument { get; set; }
        public bool Depressed { get; set; }
        public bool LowJobSatisfaction { get; set; }
    }
}

File: Frontend/src/app/sites/site-visit.ts
export class SiteVisit {

  constructor() {
    this.visitDate = new Date();
  }

  public id: number;
  public siteId: number;
  public visitDate: Date;
  public workerName: string;
  public bibleGiven: boolean;

  public teamMembers: string;
  public notes: string;
  public giftsGiven: boolean;
  public loginGiven: boolean;
  public prayer: boolean;

  public englishAbility: string;
  public age: string;
  public followUpRequired: string;

  //trafficking signs
  public dressedInappropriately: boolean;
  public inadequateLighting: boolean;
  public canNotLeave: boolean;
  public sleepingOnSite: boolean;
  public beingMovedAround: boolean;
  public doesntHaveDocuments: boolean;
  public depressed: boolean;
  public lowJobSatisfaction: boolean;
}
